### PR TITLE
Add InfluxDb

### PR DESCRIPTION
**Pull request by bion19**

### CODE REVIEW

File: InfluxDb/InfluxDb.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php namespace App\SupportedApps\InfluxDb;
```
Comment (milanmeu) [inline InfluxDb/InfluxDb.php:1]: The correct name is `InlfuxDB` please rename `InfluxDb` to `InfluxDB`.
```suggestion
<?php namespace App\SupportedApps\InfluxDB;
```

File: InfluxDb/InfluxDb.php [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+class InfluxDb extends \App\SupportedApps {
```
Comment (milanmeu) [inline InfluxDb/InfluxDb.php:3]: It is also wrong here.
```suggestion
class InfluxDB extends \App\SupportedApps {
```

File: InfluxDb/app.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "appid": "5e1e914cba38acaa83d4e39139dadbec478fbb6e",
+    "name": "InfluxDb",
```
Comment (milanmeu) [inline InfluxDb/app.json:3]: The name here is also wrong. It is correct in the app description.
```suggestion
    "name": "InfluxDB",
```

**Review by milanmeu — CHANGES_REQUESTED** [review]
The correct name is `InlfuxDB` please rename `InfluxDb` to `InfluxDB`. It is correct in the app.json description 😃. Change de folder and filenames too.

**Comment by TheCatLady** [comment]
LGTM!

**Pull request merged**